hack/mirror-origin-tests.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+. $(dirname "$0")/origin-tests-common.sh
+
+skopeo_workdir=$(mktemp -d)
+mapping_file=$(mktemp)
+function finish {
+  rm -rf ${skopeo_workdir}
+  rm -f ${mapping_file}
+}
+trap finish EXIT
+
+MIRROR_ORIGIN_TESTS_OC_TOOL="${MIRROR_ORIGIN_TESTS_OC_TOOL:-oc}"
+MIRROR_ORIGIN_TESTS_CONTAINER_MGMT_CLI="${MIRROR_ORIGIN_TESTS_CONTAINER_MGMT_CLI:-docker}"
+
+MIRROR_ORIGIN_TESTS_IN_CONTAINER="${MIRROR_ORIGIN_TESTS_IN_CONTAINER:-true}"
```

Comment:
even this one can be made more global,
setting if we want to run things in containers or not